### PR TITLE
feat: added gitops section in homepage

### LINKED ISSUE #1180: New Section for GitOps Integration
Create a new section describing GitOps integration on the Meshery website
---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by thisiskaransgit**
**Description**

This PR fixes #1180 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: _includes/home-page.html [hunk 1]
```diff
@@ -189,6 +189,31 @@ <h2><span class="special-text">Context-Aware</span> Policies For Applications</h
...
 		</div>
 	</section>
+	<section>
+		<div class="container flex">
+			<div class="text ">
+					<img src="{{ site.baseurl }}/assets/images/meshmap-icon.svg"
+						style="max-width:30%;position:relative;float:left;
+						margin-right:20px;margin-bottom: 10px;"
+						alt="Webassembly based Envoy Filters" loading="lazy"/>
+				<h2>Simplify the process of Infrastructure Managment with <span class="special-text">MeshMap</span></h2>
```
Comment (Chadha93) [inline _includes/home-page.html:199]:
```suggestion
				<h2>Simplify the process of Infrastructure Managment with <span class="special-text">Meshery Extension</span></h2>
```

Comment (Chadha93) [reply _includes/home-page.html:199]: @karnatisrinivas MeshMap is an extension of Meshery. MeshMap is a plugin of Meshery. It is built separately and loaded dynamically into Meshery UI at runtime

**Comment by vishalvivekm** [comment]
@thisiskaransgit 
Let's discuss this in websites call. Add this as agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

**Comment by Chadha93** [comment]
@thisiskaransgit Let's ensure we add relevant information and points for users to understand what and how our GitOps works, will you toss in some more short points from this page? -https://layer5.io/cloud-native-management/gitops

File: _includes/home-page.html [hunk 2]
```diff
@@ -189,6 +189,31 @@ <h2><span class="special-text">Context-Aware</span> Policies For Applications</h
...
+					<img src="{{ site.baseurl }}/assets/images/meshmap-icon.svg"
+						style="max-width:30%;position:relative;float:left;
+						margin-right:20px;margin-bottom: 10px;"
+						alt="Webassembly based Envoy Filters" loading="lazy"/>
+				<h2>Simplify the process of Infrastructure Managment with <span class="special-text">Meshery Extension</span></h2>
+				<p>Streamlining Platform Engineering with GitOps and Meshery. Preview your deployment, view changes pull request-to-pull request and get infrastructure snapshots within your PRs by connecting MeshMap to your GitHub repositories.
+					<!-- <br>
+				<div align="center" class="button-para" style="margin-top: 2rem;">
+				<a href="https://github.com/layer5io/image-hub" class="link">See Image Hub</a>
+				</div> -->
```
Comment (samyakshah3008) [inline _includes/home-page.html:204]: Curious to know why we have this commented code? 
Shall we remove it?

File: assets/images/meshmap-short-consul.gif
Comment (samyakshah3008) [inline assets/images/meshmap-short-consul.gif:1]: This gif is of 14.9 MB, can we reduce the size for better performance?

**Review by samyakshah3008 — COMMENTED** [review]
Great work @thisiskaransgit 💪 
Added couple of comments, please check. 
Rest looks good to merge into prod 🔥

**Review by Chadha93 — APPROVED** [review]
Thank you @thisiskaransgit

**Pull request merged**